### PR TITLE
Tx details

### PR DESCRIPTION
**Pull request by plondon**
- [x] improve tests
- [x] improve search filtering
- [x] finish destinations logic

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ language: node_js
 node_js:
 - 5.10
 before_install:
-- npm install -g grunt-cli semistandard snazzy
```
Comment (Sjors) [inline .travis.yml]: Why did you remove semistandard here? Is it included by snazzy?

File: .travis.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ language: node_js
...
+- npm install -g grunt-cli snazzy
 - gem install sass
 before_script:
-- semistandard --verbose | snazzy
+- node_modules/.bin/semistandard --verbose | snazzy
```
Comment (Sjors) [inline .travis.yml]: Good idea. Can we do the same with snazzy?

**Comment by Sjors** [comment]
ACK

**Comment by plondon** [comment]
@jtormey this was approved by Lars, though he suggests that we treat is as a RC.
